template/deployments/helm/charts/templates/_helpers.tpl

```diff
@@ -1,17 +1,16 @@
 {{/*
 Expand the name of the chart.
 */}}
-{{- define "{{APPNAME}}.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
-{{- end }}
+{{ printf "{{- define \"%s.name\" -}}" .APPNAME }}
+{{ printf "{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix \"-\" }}\n{{- end }}" }}
```

Comment:
Lets keep it consistent, if you dont have a draft variable replacement happening then lets just use the comment wrapping method